### PR TITLE
Fix - my profile redesign and new structure

### PR DESCRIPTION
**Pull request by dag-mara**
https://trello.com/c/5VYszPA1 & https://trello.com/c/oOs5X0zf

### CODE REVIEW

File: frontend/src/user/pages/ViewProfile/ViewProfile.tsx [hunk 1]
```diff
@@ -75,35 +102,296 @@ export const ViewProfile = () => {
...
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href="https://mozek.brontosaurus.cz/"
+                  className={styles.help}
+                >
+                  Mozek HB
+                </a>
+                , kde jsou všechny důležité dokumenty. Přihlašovací jméno:
+                brontosaurus, heslo: Brontosaurus40
```
Comment (martinapro112) [inline frontend/src/user/pages/ViewProfile/ViewProfile.tsx:344]: Seems really weird to add the password like this, not a good idea at all. But I suppose Hanka wanted that specifically.

File: frontend/src/user/pages/ViewProfile/ViewProfile.tsx [hunk 2]
```diff
@@ -75,35 +102,296 @@ export const ViewProfile = () => {
...
+                , kde můžeš využívat předplacenou Canvu Pro pro vytváření
+                plakátů a dalších propagačních materiálů. Přihlašovací email:
+                marketing@brontosaurus.cz, heslo: jedemedal24
+              </li>
+              <li>
+                Připravenou{' '}
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href="https://zpetna-vazba.brontosaurus.cz/login.php"
```
Comment (martinapro112) [inline frontend/src/user/pages/ViewProfile/ViewProfile.tsx:363]: Is the same link with just slightly different text really supposed to be there twice?

**Review by martinapro112 — APPROVED** [review]

File: frontend/src/user/pages/ViewProfile/ViewProfile.tsx (hunk 2 above)
Comment (dag-mara) [reply frontend/src/user/pages/ViewProfile/ViewProfile.tsx:363]: yes, it should be. Link is the same, but there are two separate actions that need to be done.

File: frontend/src/user/pages/ViewProfile/ViewProfile.tsx (hunk 1 above)
Comment (dag-mara) [reply frontend/src/user/pages/ViewProfile/ViewProfile.tsx:344]: She did, but I will remain her to think about it during testing.

**Pull request merged**